forms-flow-web/src/components/CustomComponents/FormInput.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Form, InputGroup } from 'react-bootstrap';
+import { Translation } from "react-i18next";
+
+const FormInput = ({
+  type = "text",
+  label,
+  value = "",
+  onChange,
+  onBlur,
+  placeholder = "",
+  isInvalid = false,
+  feedback = "",
+  disabled = false,
+  size,
+  dataTestid,
+  ariaLabel,
+  className ,
+  required = false,
+  icon,
+  id
+}) => {
+
+  const inputClassNames = "Form-control-input " + className ;
+  return (
+    <div className="Form-input-box" >
+      <Form.Group className={className} controlId={id}>
```

Comment:
please remove className here if not needed

Reply:
updated.